Dockerfile_amd

```diff
@@ -0,0 +1,85 @@
+#ARG BASE_DOCKER=rocm/pytorch:latest
+#ARG BASE_DOCKER=rocm/pytorch-private:exec_dashboard_nightly
+ARG BASE_DOCKER=rocm/pytorch-private:20240920_exec_dashboard_nightly
```

Comment:
Could you use rocm/pytorch:latest here as public users might not be able to use our internal dockers.